app/jobs/download_form_templates_job.rb

```diff
@@ -3,7 +3,7 @@ def perform(force: false)
     Heartbeat.find_or_create_by(name: self.class.name).touch
 
     eform_service = UpvsEnvironment.eform_service
-    eform_service.fetch_all_form_template_ids.each do |template_id|
+    eform_service.fetch_form_template_ids.each do |template_id|
```

Comment:
Tomuto nerozumiem, to bol celkom dobry nazov, preco sme to premenovali?

Reply:
toto si uz nepamatam, mi to asi dako padlo do oka 😅 revertnem to